sootup.analysis.interprocedural/src/main/java/sootup/analysis/interprocedural/icfg/JimpleBasedInterproceduralCFG.java

```diff
@@ -56,7 +55,7 @@ public class JimpleBasedInterproceduralCFG extends AbstractJimpleBasedICFG {
 
   protected static final Logger logger =
       LoggerFactory.getLogger(JimpleBasedInterproceduralCFG.class);
-  private final MethodSignature mainMethodSignature;
+  private List<MethodSignature> entryPoints;
```

Comment:
can't we remove that field and delegate the usages of entrypoints to cg.getEntryPoints() ?

Reply:
Yes, we can, and add a getter for the call graph so one can get all info from that :)